tests/CommonFunctionsTest.php

```diff
@@ -122,13 +122,14 @@ public function testStringToClass()
     public function testIsValidHostname()
     {
         $this->assertTrue(is_valid_hostname('a'), 'a');
-        $this->assertTrue(is_valid_hostname('a.'), 'a.');
```

Comment:
This is not a valid domain name anyway, according to me.

Reply:
It is a valid hostname however.

`example.net` and `example.net.` are both the same and valid.
The latter of this example is a `true` FQDN as it denominates the absolute root whereas the first can still imply a search-domain.